pkg/metrics/collector/sockstat_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestSockstats(t *testing.T) {
+	input := "sockets: used 270\n" +
```

Comment:
t.Parallel()

Reply:
you have multiple tests cases. Use the t.Run in the standard go test idiom:
```go
for _, tc := range []struct{
  desc string
  in string
  want socketStats   
}{
  {
    desc: "ipv4",
    ...
  },
  {
    desc: "ipv6",
    ...
  },
}{
  t.Run(tc.desc, func (t *testing.T) {
    ss, err := parseSocketStats(...)
    if err != nil {
      t.Errorf("parseSocketStats(%q) = _, %v; want _, nil", tc.in, err)
    }
    ...
  })
}


```

Reply:
This is not multiple tests. In normal flow,  output is a combination of both ipv4 and ipv6.

Reply:
make in a []string then...

Reply:
Updated.